static/app/components/group/suggestedOwners/ownershipRules.tsx

```diff
@@ -78,3 +128,39 @@ const HelpfulBody = styled('div')`
   padding: ${space(1)};
   text-align: center;
 `;
+
+const Container = styled('div')`
```

Comment:
You could probably use the `<Panel>` component here and use the dashed styles and border-radius styles from that instead of writing them custom here